evap/staff/importers/enrollment.py

```diff
@@ -336,9 +337,27 @@ def get_merge_hindrances(course_data: CourseData, merge_candidate: Course) -> Li
         merge_candidate_evaluations = merge_candidate.evaluations.all()
         if len(merge_candidate_evaluations) != 1:
             hindrances.append(_("the existing course does not have exactly one evaluation"))
-        elif merge_candidate_evaluations[0].wait_for_grade_upload_before_publishing != course_data.is_graded:
+            return hindrances
+
+        merge_candidate_evaluation: Evaluation = merge_candidate_evaluations[0]
+
+        if merge_candidate_evaluation.wait_for_grade_upload_before_publishing != course_data.is_graded:
             hindrances.append(_("the evaluation of the existing course has a mismatching grading specification"))
 
+        if merge_candidate_evaluation.is_single_result:
+            hindrances.append(_("the evaluation of the existing course is a single result"))
+            return hindrances
+
+        if merge_candidate_evaluation.state >= Evaluation.State.PUBLISHED:
```

Comment:
```suggestion
        if merge_candidate_evaluation.state >= Evaluation.State.IN_EVALUATION:
```

we shouldn't allow changes on evaluations that are already running